test/lib/CustomRulePathManager.test.ts

```diff
@@ -38,7 +41,7 @@ describe('CustomRulePathManager tests', () => {
 				const rulePathMap = await manager.getRulePathEntries(PmdEngine.NAME);
 
 				// Validate run
-				expect(readStub.calledOnce).to.be.true;
+				expect(readStub.calledTwice).to.be.true;
```

Comment:
Is it worth asserting which files should be read, the way you do at line 69-71?

Reply:
I figured it was overkill to repeat that assertion in each of the three tests with this kind of check.  I put it in 69-71 because that test seemed the most appropriate (testing it was initialized one time) vs the others which are not really about checking that.